### DISCUSSION TITLE
Feature request: Listening Statistics

### DISCUSSION THREAD

**Discussion by d4tis** [PaulWoitaschek/Voice#1308] [Ideas]
Something similar to audible or pocket casts would be great. Something simple like -

Total time listened (maybe per day/month/year)
Total books listened to

Thanks for the great app

**Comment by technomooney** [comment]
is this ever going to be talked about or planned? i am transitioning from smart audiobook player as i am trying to move away from stock google pixel / samsung.... and i dont think i can use my license for that player... but i would like to have a listening history so i can keep track of what i have listened to already. this is the only player i have found that does what i want, other then maybe the volume boost, and does not need me to self host a server for it.
